00-HTML-CSS-basics/css/2-5/2/css/styles.css

```diff
@@ -0,0 +1,86 @@
+a {
+	color: #000000;
+	font-weight: 700;
+	text-decoration: none;
+}
+
+h3 {
+	margin: 5px 0px;
+}
+
+img {
+	max-width: 100%;
+
+	-ms-filter: grayscale(1);
+	-webkit-filter: grayscale(1);
+	-moz-filter: grayscale(1);
+	-o-filter: grayscale(1);
+	filter: grayscale(1);
+
+	-webkit-transition: all 0.3s ease;                  
+    -moz-transition: all 0.3s ease;                 
+    -o-transition: all 0.3s ease;   
+    -ms-transition: all 0.3s ease;          
+    transition: all 0.3s ease;
+}
+
+figure {
+	margin: 0;
+	padding: 0;
+	position: relative;
+}
+
+figcaption {
+	background-color: rgba(255,255,255,0.9);
+	left: 0px;
```

Comment:
**Minor:** when assigning values to ``0``, never use the unit (in this case remove ``px``).